odl/discr/lp_discr.py

```diff
@@ -1257,6 +1211,9 @@ def show(self, title=None, method='', coords=None, indices=None,
             is that ``fig`` is the return value of an earlier call to
             this function.
 
+        interp : {'linear', 'nearest'}, optional
+            Interpolation type that should be used for the plot.
```

Comment:
This was simply missing in the docstring.

Reply:
Exactly. And `saveto` and `clim` are just pass-through args to `matplotlib`, so they're not really "ours".